public/index.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * Standalone GLPI web router.
+ *
+ * This router is used to be able to expose only the `/public` directory on the webserver.
+ *
+ * Requires following configuration for Apache:
+```
+<VirtualHost *:80>
+    DocumentRoot /var/www/glpi/public
+
+    [...]
+
+    <Directory /var/www/glpi/public>
+        FallbackResource /index.php
+        Require all granted
+    </Directory>
+</VirtualHost>
+```
+
+* Requires following configuration for Nginx:
+```
```

Comment:
Opened, but not closed :)

Reply:
Will have to be moved from here to the install documentation anyway.